packages/dcos-integration-test/extra/test_meta.py

```diff
@@ -0,0 +1,82 @@
+"""
+Tests for the integration test suite itself.
+"""
+
+import subprocess
+from pathlib import Path
+from typing import Set
+
+import yaml
+
+from get_test_group import patterns_from_group
+
+
+def _tests_from_pattern(ci_pattern: str) -> Set[str]:
+    """
+    From a CI pattern, get all tests ``pytest`` would collect.
+    """
+    tests = set([])  # type: Set[str]
+    args = ['pytest', '--collect-only', ci_pattern, '-q']
+    result = subprocess.run(args=args, stdout=subprocess.PIPE)
+    output = result.stdout
```

Comment:
what's with stderr emitted by this process?

Wanna say `stderr=subprocess.STDOUT` in the run() call above for a really easy way to make sure to collect both?

Reply:
Or... is it intended that `stderr` of this child process ends up straight in the stderr of the test runner?

Reply:
> Or... is it intended that stderr of this child process ends up straight in the stderr of the test runner?

Yes, there is no way that any `stderr` output is the name of a test. I will add a comment.